tests/test.unit.page.spec.ts

```diff
@@ -0,0 +1,217 @@
+import { test, expect } from "@playwright/test";
+import HomePage from '../pages/home.page';
+import CreateUnitPage from '../pages/create.unit.page'
+
+import { faker } from '@faker-js/faker';
+import { getStringWithSpaceIncide, getStringWithSpaceInEnd } from '../helpers/random_values';
+
+const validEmail: string = process.env.VALID_EMAIL || '';
+const validPassword: string = process.env.VALID_PASSWORD || '';
+const homepageUrl: string = process.env.HOMEPAGE_URL || '';
```

Comment:
It is advised to use CONSTANT_CASE for such variables. See https://ts.dev/style/#syntax for more details